forms-html/Hameed-Sara_NoorAldin/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hameed-Sara_NoorAldin</title>
+</head>
+<body>
+    
+    <form action="POST">
+        <br>
+        <label for="email">Enter your email:</label>
+        <input type="email" name="Email" placeholder="enter you email" required>
```

Comment:
The placeholder attribute specifies a short hint that describes the expected value of an input field, so it's better to be like
placeholder="example@example.com"